target/linux/ath79/image/nand.mk

```diff
@@ -211,6 +211,20 @@ define Device/glinet_gl-xe300
 endef
 TARGET_DEVICES += glinet_gl-xe300
 
+define Device/linksys_ea4500-v3
+  SOC := qca9558
+  DEVICE_VENDOR := Linksys
+  DEVICE_MODEL := EA4500
+  DEVICE_VARIANT := v3
+  DEVICE_PACKAGES := kmod-usb2
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  KERNEL_SIZE := 4096k
```

Comment:
oh, we missed `IMAGE_SIZE := 81920k` here. Everything should be fine now.

Reply:
Should I add IMAGE_SIZE := 81920k here?

Reply:
Is the current commit fine or not?

Reply:
Yes, we need to add it, it's used to check binary image size. It is equal to the size of the firmware partition. Everything else looks good to me.